Classes/ContentObject/JsonContentObject.php

```diff
@@ -14,6 +11,10 @@
  *
  ***/
 
+declare(strict_types=1);
```

Comment:
is this change intended (moving declare and namespace after comments?

Reply:
ok, looks good then

Reply:
D'no how that rule got accepted to the standard. Can only assume the reviewers missed it. I've never seen any code before that followed that rule of having the doc above the declares. The 2 authors of the PSR must have wanted it, only...